### PR TITLE
Create Qwertx.domain.json

### PR DESCRIPTION
**Pull request by NotQWertz**
I am planning to use the domain for my personal site

### Link to Website

Link:

### CODE REVIEW

**Review by tarampampam — CHANGES_REQUESTED** [review]
Please, follow the checklist

File: qwertx.is-an.app.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (tarampampam) [inline qwertx.is-an.app.json:1]: The filename should be `qwertz.is-an.spp.json` (not `qwertx.is-an.app.json`)

File: qwertx.is-an.app.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "$schema": "../schemas/domain.schema.json",
+  "description": "My personal website", 
+  "domain": "is-an.app", 
+  "subdomain": "qwertz",
+  "owner": {
+    "repo": "https://github.com/notqwertz/repo",
```
Comment (tarampampam) [inline qwertx.is-an.app.json]: <https://github.com/notqwertz/repo> doesn't exist

**Comment by NotQWertz** [comment]
Sorry fixed it hope it's now good

File: domains/Qwertx.is-an.app.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (tarampampam) [inline domains/Qwertx.is-an.app.json:1]: The filename should be `qwertx.is-an.app.json`

Comment (NotQWertz) [reply domains/Qwertx.is-an.app.json:1]: Ok

File: domains/Qwertx.is-an.app.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "$schema": "../schemas/domain.schema.json",
+  "description": "My personal site and other stuff",
+  "domain": "is-an.app",
+  "subdomain": "qwertx",
+  "owner": {
+    "repo": "https://github.com/notqwertz/free-domains",
```
Comment (tarampampam) [inline domains/Qwertx.is-an.app.json]: Is that repo with content for the subdomain?

```suggestion
    "repo": "https://github.com/NotQWertz/NotQWertz.github.io",
```

File: domains/qwertx.is-an.app.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "My personal site and other stuff",
+  "domain": "is-an.app",
+  "subdomain": "qwertx",
+  "owner": {
+    "repo": "https://github.com/notqwertz/free-domains",
```
Comment (tarampampam) [inline domains/qwertx.is-an.app.json:7]: Is that repo with your site content?

```suggestion
    "repo": "https://github.com/NotQWertz/NotQWertz.github.io",
```

**Review by tarampampam — CHANGES_REQUESTED** [review]
Make changes, please

File: domains/qwertx.is-an.app.json (hunk 5 above)
Comment (NotQWertz) [reply domains/qwertx.is-an.app.json:7]: Yeah

Comment (tarampampam) [reply domains/qwertx.is-an.app.json:7]: Update this property in this case, please :)

File: domains/qwertx.is-an.app.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    "email": "zuweyrh752@gmail.com"
+  },
+  "record": {
+    "CNAME": "notqwertz.github.io"
```
[the remark this answers is not in the document]
Comment (tarampampam) [reply domains/qwertx.is-an.app.json:11]: @NotQWertz This one

[the remark this answers is not in the document]
Comment (tarampampam) [reply domains/qwertx.is-an.app.json:11]: Ping @NotQWertz

**Comment by tarampampam** [comment]
```shell
$ dig @1.1.1.1 +noall +answer qwertx.is-an.app

qwertx.is-an.app.	300	IN	CNAME	notqwertz.github.io.
notqwertz.github.io.	3600	IN	A	185.199.108.153
notqwertz.github.io.	3600	IN	A	185.199.109.153
notqwertz.github.io.	3600	IN	A	185.199.110.153
notqwertz.github.io.	3600	IN	A	185.199.111.153
```

**Pull request merged**